docs/_eleventy/layouts/base.njk

```diff
@@ -520,10 +520,15 @@ a.grid-nav-link:hover {
 
 
 @media (max-width: 860px) {
+  .grid-body-header {
+    margin: 3rem 0 0;
+  }
   .markdown-body, .grid-body-header, .toc {
     padding: 0 1em;
   }
-
+.grid-nav-container {
+    width: 100vw;
```

Comment:
👍🏻 

`100%` never does what you think it will, does it? 😄

Reply:
haha sadly, no